### PR TITLE
add highlightValue

### PR DESCRIPTION
**Pull request by idealllee**
@jlo1

### CODE REVIEW

File: android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java [hunk 1]
```diff
@@ -382,5 +382,15 @@ public void setData(Chart chart, ReadableMap propMap) {
...
-
+    /**
+     * details: https://github.com/PhilJay/MPAndroidChart/wiki/Highlighting
+     */
+    @ReactProp(name = "highlightValue")
+    public void setHighlightValue(Chart chart, ReadableMap propMap) {
+        chart.highlightValue(
+            (float) propMap.getDouble("x"),
+            (int) propMap.getInt("dataSetIndex"),
+            (boolean) propMap.getBoolean("callListener")
```
Comment (drhops) [inline android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java]: out of curiosity, why do these need to be cast if the function name is specific to the type?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/indiemoney/react-native-charts-wrapper"
   },
-  "version": "0.2.11",
+  "version": "0.2.12",
```
Comment (drhops) [inline package.json:7]: are we doing PRs upstream too?

**Comment by drhops** [comment]
LGTM :scorpion:

File: android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java (hunk 1 above)
Comment (idealllee) [reply android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java]: good catch. we don't need to convert (except for double->float). will update

File: package.json (hunk 2 above)
Comment (idealllee) [reply package.json:7]: been doing it but the author isn't very responsive: https://github.com/wuxudong/react-native-charts-wrapper/pulls

**Pull request merged**